src/styles/StudentStyles/index.less

```diff
@@ -144,6 +146,108 @@
     border-radius: 25px;
 }
 
+.student-story-card h2 {
+    font-size: 1.8rem;
+}
+
+.student-story-card h4 {
+    font-size: 1.2rem;
+}
+
 .student-story-card img {
     border-radius: 100%;
+}
+
+.top-story-card {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.top-left {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 250px;
+}
+
+.top-left img {
+    height: 50px;
+}
+
+.top-left h3 {
+    text-transform: capitalize;
+}
+
+.top-left h4 {
+    text-transform: capitalize;
+}
+
+.top-right {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.top-right h4 {
+    color: grey;
+    text-transform: uppercase;
+}
+
+.teacher-message {
+    padding: 5% 10% 2%;
+}
+
+.story-photo {
+    background-image: url('https://cdn.pixabay.com/photo/2016/08/03/09/03/universe-1566159_1280.jpg');
+    height: 100px;
+    width: 100%;
+}
+
+/* Story Card Comment */
+
+.student-comment {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 5%;
+}
+
+.student-comment-info {
+    display: flex;
+    flex-direction: row;
+    width: 350px;
+}
+
+.student-comment-info img {
+    height: 50px;
+}
+
+.student-comment-info div {
+    padding: 0% 5%;
+}
+
+.student-comment-info div p {
+    font-size: 12px;
+}
+
+.comment-minute h4 {
+    color: grey;
+}
+
+.student-send-comment {
+    padding-top: 5%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.student-send-comment form input{
+    width: 400px;
+    border-radius: 5px;
+    height: 45px;
+}
+
+.comment-form {
+    display: flex;
+    justify-content: space-between;
```

Comment:
Leave a new line at the end of the file.

Reply:
Just fixed this. Thanks!